packages/flutter/test/material/color_scheme_test.dart

```diff
@@ -9,76 +9,156 @@ void main() {
   test('light scheme matches the spec', () {
     // Colors should match the Material Design baseline default theme:
     // https://material.io/design/color/dark-theme.html#ui-application
+    // with the new Material 3 colors defaulting to values from the M2
+    // baseline.
     const ColorScheme scheme = ColorScheme.light();
+    expect(scheme.brightness, Brightness.light);
     expect(scheme.primary, const Color(0xff6200ee));
-    expect(scheme.primaryVariant, const Color(0xff3700b3));
-    expect(scheme.secondary, const Color(0xff03dac6));
-    expect(scheme.secondaryVariant, const Color(0xff018786));
-    expect(scheme.background, const Color(0xffffffff));
-    expect(scheme.surface, const Color(0xffffffff));
-    expect(scheme.error, const Color(0xffb00020));
     expect(scheme.onPrimary, const Color(0xffffffff));
+    expect(scheme.primaryContainer, scheme.primary);
+    expect(scheme.onPrimaryContainer, scheme.onPrimary);
+    expect(scheme.secondary, const Color(0xff03dac6));
     expect(scheme.onSecondary, const Color(0xff000000));
+    expect(scheme.secondaryContainer, scheme.secondary);
+    expect(scheme.onSecondaryContainer, scheme.onSecondary);
+    expect(scheme.tertiary, scheme.secondary);
+    expect(scheme.onTertiary, scheme.onSecondary);
+    expect(scheme.tertiaryContainer, scheme.tertiary);
+    expect(scheme.onTertiaryContainer, scheme.onTertiary);
+    expect(scheme.error, const Color(0xffb00020));
+    expect(scheme.onError, const Color(0xffffffff));
+    expect(scheme.errorContainer, scheme.error);
+    expect(scheme.onErrorContainer, scheme.onError);
+    expect(scheme.background, const Color(0xffffffff));
     expect(scheme.onBackground, const Color(0xff000000));
+    expect(scheme.surface, const Color(0xffffffff));
     expect(scheme.onSurface, const Color(0xff000000));
-    expect(scheme.onError, const Color(0xffffffff));
-    expect(scheme.brightness, Brightness.light);
+    expect(scheme.surfaceVariant, scheme.surface);
+    expect(scheme.onSurfaceVariant, scheme.onSurface);
+    expect(scheme.outline, scheme.onBackground);
+    expect(scheme.shadow, scheme.onBackground);
+    expect(scheme.inverseSurface, scheme.onSurface);
+    expect(scheme.onInverseSurface, scheme.surface);
```

Comment:
FYI, design is going with `inverseOnSurface` (see b/205944640). Let's make sure the documentation for it mentions that it's the color to use on `inverseSurface`